drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -294,6 +294,7 @@ void RasterizerGLES3::_blit_render_target_to_screen(RID p_render_target, Display
 		flip_y = false;
 	}
 
+
```

Comment:
Please make sure not to include unnecessary changes in files with no functional change.